### DISCUSSION TITLE
Update source using generate command

### DISCUSSION THREAD

**Discussion by danikiev** [idaholab/moose#29105] [Q&A Tools] [answered]
I want to generate source markdowns for a new object only using ` ./moosedocs.py generate` command.
However it recreates markdown files even for all objects. How to avoid it and update only the specific object?

**Comment by GiudGiud** [comment, marked as answer]
Hello

There is no option at the moment to do that officially. 
You could hack it by :
- changing the app_type of the one object
- recompile
- select the app_type using the `--app_types` after `./moosedocs.py generate`
- change the app_type back for the object

but at this point you might as well copy paste an existing documentation page and change the object name

**Reply by danikiev** [reply]
Thanks for the hints!
